metadata_mapper/lambda_function.py

```diff
@@ -17,16 +18,19 @@ def import_vernacular_reader(mapper_type):
     nuxeo       | nuxeo_mapper        | NuxeoVernacular
     content_dm  | content_dm_mapper   | ContentDmVernacular
     """
+    mapper_parent_modules, snake_mapper_name = mapper_type.split(".")
+
     mapper_module = importlib.import_module(
-        f"mappers.{mapper_type}_mapper", package="metadata_mapper")
+        f"mappers.{mapper_parent_modules}.{snake_mapper_name}_mapper",
+        package="metadata_mapper")
 
-    mapper_type_words = mapper_type.split('_')
+    mapper_type_words = snake_mapper_name.split('_')
     class_type = ''.join([word.capitalize() for word in mapper_type_words])
     vernacular_class = getattr(
         mapper_module, f"{class_type}Vernacular")
 
-    if vernacular_class not in VernacularReader.__subclasses__():
-        print(f"{ mapper_type } not a subclass of VernacularReader")
+    if not issubclass(vernacular_class, AbstractVernacular):
```

Comment:
A simple issubclass won't quite work here (IslandoraVernacular is a subclass of OaiVernacular is a subclass of AbstractVernacular), but I've added a recursive implementation `all_subclasses` here: https://github.com/ucldc/rikolti/pull/244/files (along with heaps of bug fixes in the enrichments)

Reply:
`issubclass` looks down the entire ancestor chain for direct, indirect, or virtual subclasses. To the best of my understanding, `__subclasses__()` only looks at direct descendents, which is why I made the change since our mapper implementations (such as `CcaVaultVernacular`) are now separated by two levels.